### PR TITLE
FAT-203-test-plan-for-mod-template-engine-added

### PR DESCRIPTION
**Pull request by Mykyta-Varenyk**
Test plan for mod-template-engine added.
https://issues.folio.org/browse/FAT-203

### CODE REVIEW

File: mod-template-engine/src/main/resources/domain/mod-template-engine/features/templateEngine.feature [hunk 1]
```diff
@@ -8,4 +8,64 @@ Feature: Template engine
   Scenario: Get all templates
     Given path 'templates'
     When method GET
-    Then status 200
\ No newline at end of file
+    Then status 200
+
```
Comment (OleksandrVidinieiev) [inline mod-template-engine/src/main/resources/domain/mod-template-engine/features/templateEngine.feature:12]: Please split these tests into separate feature-files by endpoint. Something like:
`templates.feature` for requests to `/templates`
`templateRequest.feature` for requests to `/template-request`

Take tests for `mod-feesfines` as an example.

Comment (Mykyta-Varenyk) [reply mod-template-engine/src/main/resources/domain/mod-template-engine/features/templateEngine.feature]: Fixed.

File: mod-template-engine/src/main/resources/domain/mod-template-engine/features/templateRequest.feature [hunk 2]
```diff
@@ -0,0 +1,26 @@
+Feature: Template processing requests tests
+
+  Background:
+    * url baseUrl
+    * callonce login testUser
+    * configure headers = { 'Content-Type': 'application/json', 'x-okapi-token': '#(okapitoken)', 'Accept': 'application/json, text/plain' }
```
Comment (OleksandrVidinieiev) [inline mod-template-engine/src/main/resources/domain/mod-template-engine/features/templateRequest.feature:6]: Not necessary at this point

**Review by OleksandrVidinieiev — APPROVED** [review]

File: mod-template-engine/src/main/resources/domain/mod-template-engine/features/templateRequest.feature (hunk 2 above)
Comment (Mykyta-Varenyk) [reply mod-template-engine/src/main/resources/domain/mod-template-engine/features/templateRequest.feature]: Fixed.

**Pull request merged**